Makefile

```diff
@@ -267,28 +269,28 @@ images: k8s-broker-image user-broker-image \
 k8s-broker-image: contrib/build/k8s-broker/Dockerfile $(BINDIR)/k8s-broker
 	mkdir -p contrib/build/k8s-broker/tmp
 	cp $(BINDIR)/k8s-broker contrib/build/k8s-broker/tmp
-	docker build -t k8s-broker:$(VERSION) contrib/build/k8s-broker
+	docker build -t $(IMAGES_PREFIX)k8s-broker:$(VERSION) contrib/build/k8s-broker
 	rm -rf contrib/build/k8s-broker/tmp
 
 user-broker-image: contrib/build/user-broker/Dockerfile $(BINDIR)/user-broker
 	mkdir -p contrib/build/user-broker/tmp
 	cp $(BINDIR)/user-broker contrib/build/user-broker/tmp
-	docker build -t user-broker:$(VERSION) contrib/build/user-broker
-	docker tag user-broker:$(VERSION) user-broker:latest
+	docker build -t $(IMAGES_PREFIX)user-broker:$(VERSION) contrib/build/user-broker
+	docker tag $(IMAGES_PREFIX)user-broker:$(VERSION) $(IMAGES_PREFIX)user-broker:latest
```

Comment:
yes, although I do think that we should go with #529 (shameless plug, sorry) because it includes the functionality to push automatically to a public repository